### ISSUE TITLE
Alternative Input for Leg Swing and Arm Swing - double check

### ISSUE THREAD

**Issue by terracoda** [phetsims/john-travoltage#425]
### Notes from Slack
**Using my keyboard**
- JT's Arm swing: 
    - Up Arrow and Right Arrow move hand clockwise; 
    - Down Arrow and Left Arrow move hand counter-clockwise
- JT's leg swing: 
    - Up Arrow and Right Arrow move foot counter-clockwise; 
    - Down Arrow and Left Arrow move foot clockwise

**Using swipes on my iphone 8**
Note that swipes left and right move focus between items when VO is enabled.
- JT's Arm swing: 
    - Swipes up move hand clockwise; 
    - Swipes down move hand counter-clockwise.
- JT's Leg swing: 
    - Swipes up move foot counter-clockwise; 
    - Swipes down move foot clockwise

I know there is something special about the direction of the Arm swing. I think we intentionally went with making the Right and Left Arrow keys move the hand in the most intuitive visual direction (right clockwise; left counter-clockwise). This means that Up Arrow moves clockwise and Down Arrow moves counterclockwise which is less intuitive.

On iOS with VO enabled, UP and Down Swipes move sliders. Right and Left swipes move focus. 

So, on iOS the Up Swipe moves the hand down (clockwise) and the Down Swipe moves the hand up (counter-clockwise).

@emily-phet, and @BLFiedler are you both fine that Up Swipes moving the arm down and Down Swipes moving the arm up?

**Comment by emily-phet** [comment]
I think however we arrange this, there will always be at least one user scenario that has an unintuitive outcome (spatially).
If we could flip the mappings for up arrow and down arrow, and flip the mappings for the up swipe and down swipe, would that be preferable across the board?

So the mappings would then be:
Up Arrow - Counter Clockwise
Up Swipe - Counter Clockwise
L Arrow - Counter Clockwise 

Down Arrow - Clockwise
Down Swipe - Clockwise
R Arrow - Clockwise 

I believe at some point, long ago, the position values were read out, so the incrementing/decrementing position values was part of what was read out. I don't think that's the case anymore, so it's just the spatial mapping read out.

**Comment by emily-phet** [comment]
Also note - I'm sure if asked @BLFiedler might have an opinion, but I don't think he's ever taken on the role of designer for this sim. I think it's ultimately up to you @terracoda.

**Comment by jessegreenberg** [comment]
@terracoda and I met to discuss this issue because it could have an impact on #391. In the end we decided to do nothing for #391, but that I would take a pass at this change to see if it would be tricky. Turned out to be pretty simple and I did it in the above commit. Can be reverted, but I made the commit so we can play with the change.

Here is a version with the arm motion reversed:
https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-dev.43/phet/john-travoltage_en_phet.html

**Comment by terracoda** [comment]
@jessegreenberg and I discussed this issue briefly when we were looking at #391.

Recently, I realized that the issue with the arm and the leg could stem from the fact that the Arm Swing, from its starting position, seems like a vertical slider whereas the Leg Swing has a more natural fit as a horizontal slider.

Neither, however, are flat-line interactions, the Arm Swing follows a full circle, and the Leg Swing follows a half-circle.

At design meeting today, we can discuss the pros and cons of adding explicit orientation to either or both of these sliders, or adding some more explicit information in the keyboard help dialog to make it more clear what actions make the appendages move forward and backward or clockwise and counter-clockwise.

Currently, the Arm Swing has the opposite orientation when compared to the Leg Swing:
- Arm Swing moves 
    - Clockwise with Right Arrow and Up Arrow
    - Counter-clockwise with Left Arrow and Down Arrow
- Leg Swing moves 
    - Counter-clockwise with Right Arrow and Up Arrow
    - Clockwise with Left Arrow and Down Arrow

**Comment by terracoda** [comment]
Thanks @jessegreenberg for https://github.com/phetsims/john-travoltage/issues/425#issuecomment-879228551

I'll try it out.

**Comment by terracoda** [comment]
I personally like it. 
I don't hear any additional slider orientation language, so I am assuming none was added.

This is what I like:
- The Leg and Arm are now mapped to the same directions in terms of clockwise and counter-clockwise
- For the Leg Swing - Movement of foot 
    - using any Arrow key seems correct and intuitive
    - using Up and Down swipes seem correct and intuitive
- For the Arm Swing - Movement of hand
    - from starting position and above the doorknob, the Up and Down Arrow keys and Up and Down swipes feel correct. The Arm moves up and down or in other words backwards and forwards around the circle.
    - After interaction, things may get a little less natural and intuitive, but I feel it is better now. I like using Up and Down Arrows when above the doorknob and Left and Right Arrow keys below the 
    - I like having the Up and Down swipes show the natural up and down direction on mobile with VoiceOver enabled.

**Comment by terracoda** [comment]
In Voicing design meeting, July 13, we decided to permanently change the direction of the Arm Swing slider so that it goes in the same direction as the Leg Swing slider. This change affects the sim, not a single feature (i.e., Voicing versus PDOM).

This means that 
- the Up and Down swipes on iOS VoiceOver now map nicely to the visuals.
- both appendages move in the same direction when using he Arrow keys.
- the Up and Down Arrows keys map nicely to the Arm Swing's initial starting position.

We felt this change required an update to the Keyboard Shortcuts dialog.

Which is discussed laid out in the [Interactive Description Design Document](https://docs.google.com/document/d/1mDGOMe1fWC4Sdz8dpA1ehyBmuUh4SCkqpGfsUGfAPA0/edit#heading=h.xj9b59zfy2x8), and in a new related issue.

**Comment by terracoda** [comment]
@jessegreenberg, I think it is safe to close this issue. Please re-open if you need to.
